pdc/apps/package/serializers.py

```diff
@@ -139,6 +140,16 @@ def to_internal_value(self, data):
                                               rpm.id,
                                               'null',
                                               json.dumps(rpm.export()))
+
```

Comment:
The function is pretty large with big cyclomatic complexity. Can you do something about it?

Reply:
Do you have a tool to measure the complexity? (In this case it's obvious, but having something automatic would be nice.)

Reply:
Not for Python - using only my eyes - though there must some other tool :).